src/shared/constants/pageOptions.ts

```diff
@@ -1 +1 @@
-export const PAGE_OPTIONS = [9, 18, 36];
+export const PAGE_OPTIONS = [9, 18, 36, 24];
```

Comment:
Why do u add option "24"?
It is used for catalog page

Reply:
i used it in IconBlock as dimensions of icon

Reply:
Create another const variable. Don`t change this one

Reply:
fixed